dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -393,7 +400,7 @@ export const LiveLayout = ({ CAPIArticle, NAV, format }: Props) => {
 
 			<main>
 				<article>
-					{CAPIArticle.matchUrl ? (
+					{CAPIArticle.matchUrl && isFootballMatch ? (
```

Comment:
You need to keep the `CAPIArticle.matchUrl` check here otherwise TS doesn't consider it to `string` instead of `string | undefined` (a weird limitation of TS in my opinion)